### DISCUSSION TITLE
Strange issue with categories in left-sidebar (autogenerated and custom)

### DISCUSSION THREAD

**Discussion by firasrg** [facebook/docusaurus#8854] [Q&A] [answered]
Hello Docusarus!

I'm adding some docs inside ``/docs`` directory for some purposes, but there is a problem when I add a 3rd level! Knowing that I see its working on your website (e.g: [**Guides** category](https://docusaurus.io/fr/docs/category/guides)).

I've used the playground on codesandbox to make [a small example](https://codesandbox.io/p/sandbox/reverent-goodall-5yvuk4?selection=%5B%7B%22endColumn%22%3A1%2C%22endLineNumber%22%3A1%2C%22startColumn%22%3A1%2C%22startLineNumber%22%3A1%7D%5D) (Its demonstrated in the demo below). 

So am I doing any mistake here or is it bug or something else ? 

Please help

## Demo:
![issue-sidebar-demo-5](https://user-images.githubusercontent.com/37019463/229084807-dd7433cf-f1b3-49d6-a5ad-2e828141d36f.gif)

**Comment by firasrg** [comment]
Hello!

After paying close attention to my source files, I've realized that I'm doing a mistake! It seems the missing _category_.json file is behind the issue!

**Comment by firasrg** [comment]
It seems that it's not related to _category_.json file. I think there is a bug! I still wait for any answer here. Please help 😅

**Comment by slorber** [comment]
Your sandbox looks fine to me 🤷‍♂️ 

![CleanShot 2023-04-05 at 18 39 06@2x](https://user-images.githubusercontent.com/749374/230147062-32b5c80d-af6d-4f80-bcad-5cf09eb1e0c3.png)

**Reply by firasrg** [reply]
Hello! 

thanks for your reply! well it seems that the issue only appears on some machines. I'm using Windows 10 Pro version 22H2 (recent), Google Chrome (latest) and Microsoft Edge browsers. maybe it's related to Operating system or maybe to node version (mine is v16). 

I've discussed with some people on Discord, and it seems that someone had same behavior. What do you think?

**Reply by slorber** [reply]
I don't know, I can't reproduce so it's hard to help.

Show me the console logs.

And try to inspect the dom structure with browser tools to see what happens. Are the child elements added to the dom? Are the classes updated when you click on the button?

**Reply by firasrg** [reply, marked as answer]
I solved it ! The issue was cause of **a performance settings** on Windows known as [Performance options](https://computerinfobits.com/adjust-visual-effects-in-windows-10/#:~:text=effects%20options%20window.-,3.%20Configure%20the%20visual%20effects%20options,-If%20your%20computer). 

The options were all unchecked (because I needed to increase performance once ago) 
![image](https://user-images.githubusercontent.com/37019463/230868483-fe5dcda1-2bf0-4813-aa08-4eb4a2e889ac.png)

After cheking the [Animate..] option, the issue has disappeared, now I see the navigation on sidebar working properly :
![docu-side-bar-fixed](https://user-images.githubusercontent.com/37019463/230868964-89bfe21f-b966-4b2b-8269-2382bdd88e90.gif)

**Reply by slorber** [reply]
This is a bug, not a windows only problem.

See https://github.com/facebook/docusaurus/discussions/8905#discussioncomment-5663928

**Reply by slorber** [reply]
https://github.com/facebook/docusaurus/pull/8906
